snapcraft/_baseplugin.py

```diff
@@ -75,11 +75,6 @@ def schema(cls):
             'build-properties': ['disable-parallel']
         }
 
-    @property
-    def PLUGIN_STAGE_SOURCES(self):
```

Comment:
> we just had a talk about breaking API

Fair point! Honestly in my head this wasn't part of the public API, but you're right, it is. I wouldn't say _PLUGIN_STAGE_SOURCES is public though, would you?